protocols/00f7b1_part10/00f7b1_part10.ot2.apiv2.py

```diff
@@ -0,0 +1,343 @@
+# flake8: noqa
```

Comment:
@john-c-lynch @ramifarawi can you remove these going forward? We want there to be QA on the protocols to ensure that they'll upload to the robot